### PR TITLE
Solutions to first exercise.

### PR DESCRIPTION
**Pull request by jdenholm**

### CODE REVIEW

**Comment by jdenholm** [comment]
Hey folks, I have pushed a first-pass at these solutions. Please do review them—and the more the merrier (or more useful).

File: exercise-solutions/01_penguin_classification_solutions.ipynb [hunk 1]
```diff
@@ -396,35 +401,55 @@
...
+    "        Returns\n",
+    "        -------\n",
+    "        Tensor\n",
+    "            The result of passing ``batch`` through the model.\n",
+    "\n",
+    "        \"\"\"\n",
+    "        return self._fwd_seq(batch)\n",
+    "\n",
+    "\n",
+<<<<<<< HEAD
```
Comment (MarionBWeinzierl) [inline exercise-solutions/01_penguin_classification_solutions.ipynb:451]: Leftovers from merge

Comment (jatkinson1000) [reply exercise-solutions/01_penguin_classification_solutions.ipynb]: Fixed this in below commit.

**Comment by MarionBWeinzierl** [comment]
I would suggest to clear the outputs of the notebooks (in kernel->restart kernel and clear outputs or something like that) before committing. Not essential, but I find it a bit cleaner.

**Comment by jdenholm** [comment]
> happy



> I would suggest to clear the outputs of the notebooks (in kernel->restart kernel and clear outputs or something like that) before committing. Not essential, but I find it a bit cleaner.

Done :)

**Review by jatkinson1000 — APPROVED** [review]
Run these and all LGTM.
I'll merge shortly :)
Thanks!

**Pull request merged**